test/module/ForwardFlow.Module.specs.ts

```diff
@@ -243,45 +243,13 @@ describe("NEW::: Forward Flow Module", async () => {
     
   });
 
+  /*
   it("Can process Personal-signed txn", async () => {
```

Comment:
personal_sign can still be used right? do we have supporting test cases for this change, positive and negative(eth_sign)

Reply:
I'll add the mentioned test cases with the next commit